internal/commands/tunnel_up.go

```diff
@@ -523,3 +529,21 @@ func setAWSCredentials(sess *session.Session) error {
 
 	return nil
 }
+
+// Execute command and return exited code.
+func execCmd(cmd *exec.Cmd) (string, int) {
+	var waitStatus syscall.WaitStatus
```

Comment:
Does this function read stderr as well? Looks like only stdout

Does it needs something like this:
```
var outbuf, errbuf strings.Builder // or bytes.Buffer
cmd.Stdout = &outbuf
cmd.Stderr = &errbuf

stdout := outbuf.String()
stderr := errbuf.String()
```